zulipterminal/model.py

```diff
@@ -282,8 +282,8 @@ def _update_initial_data(self) -> None:
                 )
             except zulip.ZulipError as e:
                 raise ServerConnectionFailure(e)
-            except Exception:
-                print("Invalid API key")
+            if response['result'] == 'error':
+                print("\n\033[91m" + response['msg'] + "\033[0m")
```

Comment:
I didn't change it when I added the other exception case, since this error handling was already present, but we should probably handle the error in core.py or run.py, with the `ServerConnectionFailure` perhaps?

It feels like a bit of a 'code smell' to have `urwid` referenced here.

Reply:
The `client` (i.e. python-zulip-api) doesn't crash unless there's no internet from my past experience. So, I think we can let this pass.